File: Level_1/x만큼 간격이 있는 n개의 숫자/solution.java
class Solution {
    public long[] solution(long x, int n) {
        long[] answer = new long[n];
        long j = 1;
        
        for(int i = 0; i < answer.length; i++, j++){
            answer[i] = x * j;
        }
        
        return answer;
    }
}

File: Level_1/x만큼 간격이 있는 n개의 숫자/solution.cpp
#include <string>
#include <vector>

using namespace std;

vector<long long> solution(int x, int n) {
    vector<long long> answer(n);
    long cnt = 1;
    
    for(int i = 0; i < answer.size(); i++, cnt++){
        answer[i] = x * cnt;
    }
    
    return answer;
}
